test/runner/requirements/sanity.txt

```diff
@@ -8,4 +8,4 @@ pytest
 rstcheck
 sphinx
 voluptuous
-yamllint
+yamllint >= 1.8.1
```

Comment:
This should not be needed with the above requested change. Also, we keep version constraints in the `constraints.txt` file so they can be shared across multiple requirements files.